rest/utilities_testing.go

```diff
@@ -46,8 +46,27 @@ type RestTester struct {
 	InitSyncSeq             uint64 // If specified, initializes _sync:seq on bucket creation.  Not supported when running against walrus
 }
 
+type RestTester struct {
+	*RestTesterConfig
+	Testing testing.TB
+}
+
+func NewRestTester(Tester testing.TB, RestConfig *RestTesterConfig) *RestTester {
+	var rt RestTester
+	rt.Testing = Tester
```

Comment:
We probably want to check that tester isn't `nil` to prevent `NewRestTester(nil, nil)`